README_MANUAL.md

```diff
@@ -115,6 +115,8 @@ Now start the ovnkube utility on the master node.
 The below command expects the user to provide
 * A cluster wide private address range of $CLUSTER_IP_SUBNET
 (e.g: 192.168.0.0/16).  The pods are provided IP address from this range.
+Note that with IPv6, there’s a limitation that the CIDR prefix must only have
+the first two bytes set.  For example: fd01::/48.
```

Comment:
Oh, sorry, I didn't realize that this applied to the cluster network... I was thinking it was only for the ovn-kube-private internal IPs. That's a little more dubious then...

Reply:
I didn't think so either.  That's not where I hit the problem.  When I was adding validation, I went back and traced everywhere it's used and realized it's used on the cluster network, too.  This is definitely far from ideal.  It makes it work, but we should think of something better.  The current patch at least unblocks us.